src/to-be-disabled.js

```diff
@@ -52,7 +56,10 @@ function isAncestorDisabled(element) {
 export function toBeDisabled(element) {
   checkHtmlElement(element, toBeDisabled, this)
 
-  const isDisabled = isElementDisabled(element) || isAncestorDisabled(element)
+  const isDisabled =
+    hasAriaDisabled(element) ||
+    isElementDisabled(element) ||
```

Comment:
I wonder if there's some issue when having conflicting signals in an element that has different values for `disabled` and `aria-disabled`. Not saying we need to do anything about it, just wondering.